### PR TITLE
Login and Register

### PR DESCRIPTION
**Pull request by antoniliev31**
Login and Register

### CODE REVIEW

**Review by antoniliev31 — COMMENTED** [review]
Login and Register

File: AI-Social-Platform.Common/NotificationMessagesConstants.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AI_Social_Platform.Common
+{
+    public static class NotificationMessagesConstants
+    {
+        public const string ErrorMessage = "Expected Error!";
```
Comment (djenitoo) [inline AI-Social-Platform.Common/NotificationMessagesConstants.cs:11]: "Expected" or "Unexpected" error?

File: AI-Social-Platform.Server/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=DESKTOP-GIE8HH7\\SQLEXPRESS;Database=AI-Social-Platform;Trusted_Connection=True;MultipleActiveResultSets=True"
```
Comment (djenitoo) [inline AI-Social-Platform.Server/appsettings.json:3]: Please add comment in file in possible - that the connection string(server) should be changed by each teammate when they get the changes. I don't remember if json file allowed comments - if not - please write it in the communication channel and tag everyone.

File: AI-Social-Platform.Common/NotificationMessagesConstants.cs (hunk 1 above)
Comment (antoniliev31) [reply AI-Social-Platform.Common/NotificationMessagesConstants.cs:11]: Unexpected! I fixed it.

File: AI-Social-Platform.Server/appsettings.json (hunk 2 above)
Comment (antoniliev31) [reply AI-Social-Platform.Server/appsettings.json:3]: Sorry, my mistake. I fixed it.

**Review by djenitoo — APPROVED** [review]

**Pull request merged**